samples/csharp/MicrosoftGraphSDK/Submission.cs

```diff
@@ -157,5 +188,135 @@ public class Submission
                 throw new GraphException($"ReassignAsync call: {ex.Message}", ex, classId, assignmentId, submissionId);
             }
         }
+
+        /// <summary>
+        /// Creating Feedback resource outcome
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <param name="submissionId">Student submission id</param>
+        /// <returns>EducationOutcome</returns>
+        public static async Task<EducationOutcome> CreateFeedbackResourceOutcome(
```

Comment:
CreateFeedbackResourceOutcomeAsync
Can we do a consistency check across the board to make sure all async methods are named correctly?

Reply:
updated "CreateFeedbackResourceOutcome" to "CreateFeedbackResourceOutcomeAsync" and checked for consistency across the solution.